src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -58,7 +62,28 @@ const MemberListItem = ({
     member.ParticipationDescription,
   );
   const [participation, setParticipation] = useState(member.Participation);
-  const [titleComment, setTitleComment] = useState(member.Description);
+  const [title, setTitle] = useState(member.Description);
```

Comment:
I'm guessing you renamed this for clarity (since it's now being used just for a person's title in a group)?